odoo/addons/base/views/ir_profile_views.xml

```diff
@@ -43,6 +43,30 @@
         </field>
     </record>
 
+    <record id="enable_profiling_wizard" model="ir.ui.view">
+        <field name="name">Enable profiling</field>
+        <field name="model">base.enable.profiling.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Enable profiling">
+                <div class="alert alert-warning" role="alert">
+                    <h3>Profiling is currently disabled on this database.</h3>
+                    Profiling is a developer feature that should be used with caution on production database.
+                    Please don't activate this feature without knowing what you are doing.
+                    Enabling profiling here will allow all user to activate profiling on their session.
+                    Profiling can be disabled at any moment in the settings.
```

Comment:
```suggestion
                    <h3>Profiling is currently disabled.</h3>
                    Profiling is a developer feature that should be used with caution.
                    It may add some load on the server, and potentially make it less responsive.
                    Enabling the profiling here allows all users to activate profiling on their session.
                    Profiling can be disabled at any moment in the settings.
```